lib/tmuxinator/assets/template.erb

```diff
@@ -48,8 +37,8 @@ cd <%= root || "." %>
     <% end %>
   <% end %>
 
-  # Create other windows.
-  <% windows.drop(1).each do |window| %>
+  # Create windows.
+  <% windows.each do |window| %>
```

Comment:
Using `new-window` with the `-k` option allows us to treat the first window, which is created when we create the session, like all the other windows.

Reply:
Clean